addons/payment/wizards/payment_capture_wizard.py

```diff
@@ -0,0 +1,153 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+from odoo.tools import format_amount
+
+
+class PaymentCaptureWizard(models.TransientModel):
+    _name = 'payment.capture.wizard'
+    _description = "Payment Capture Wizard"
+
+    transaction_ids = fields.Many2many(
+        comodel_name='payment.transaction',
+        string="Parent transactions that are or were authorized",
+        default=lambda self: self.env.context.get('active_ids'),
+        readonly=True,
+    )
+    authorized_amount = fields.Monetary(
+        string="Authorized Amount", compute='_compute_authorized_amount',
+    )
+    amount_available = fields.Monetary(
+        string="Maximum Capture Allowed", compute='_compute_amount_available',
+    )
+    captured_amount = fields.Monetary(string="Already Captured", compute='_compute_captured_amount')
+    voided_amount = fields.Monetary(string="Already Voided", compute='_compute_voided_amount')
+    amount_to_capture = fields.Monetary(
+        string="Capture Amount", compute='_compute_amount_to_capture', store=True, readonly=False,
+    )
+    currency_id = fields.Many2one(string="Currency", related='transaction_ids.currency_id')
+    # TODO edm ask anv: is this okay with ids instead of id? We know it's the same, but what would
+    #  happens if it wasn't?
+    support_partial_capture = fields.Boolean(
+        compute='_compute_support_partial_capture',
+        help="Whether all the tx are from a provider that support the partial capture",
+        compute_sudo=True,
+    )
+    has_draft_children = fields.Boolean(
+        string="Has a draft children", compute='_compute_has_draft_children',
+    )
+    has_remaining_amount = fields.Boolean(
+        string="Has Remaining Amount", compute='_compute_has_remaining_amount',
+    )
+    void_remaining_amount = fields.Boolean(string="Void remaining amount")
+
+    #=== COMPUTE METHODS ===#
+
+    @api.depends('transaction_ids')
+    def _compute_authorized_amount(self):
+        for wizard in self:
+            wizard.authorized_amount = sum(wizard.transaction_ids.mapped('amount'))
```

Comment:
Nop, multiple partial captures, the source tx might have child but still needs to be counted in =]
Or even being fully processed in children, being in state "done", and still needs to be counted in the authorized amount

Reply:
And as it is, only parent transaction are passed in the wizard, so that's not an issue since I took off the `invoices_ids` on the child tx =]

Reply:
Indeed, my suggestion should have been:
```suggestion
            wizard.authorized_amount = sum(wizard.transaction_ids.filter(lamba tx: not tx.source_transaction_ids).mapped('amount'))
```

But it's okay like this since it should not be possible to get child transactions in this wizard, and as we discussed, it will be a good way to find out if there is a problem in that flow